README.md

```diff
@@ -37,28 +36,30 @@ Now access Dashboard at:
 [`http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/`](
 http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/).
 
-To find out how to create sample user and log in follow [Creating sample user](https://github.com/kubernetes/dashboard/wiki/Creating-sample-user) guide.
+To find out how to create sample user and log in follow [Creating sample user](docs/Creating-sample-user.md) guide.
 
 **NOTE:**
 * The shortcut `http://localhost:8001/ui` is deprecated. Use the full proxy URL shown above.
 * [Heapster](https://github.com/kubernetes/heapster/) has to be running in the cluster for the metrics
-and graphs to be available. Read more about it in [Integrations](
-https://github.com/kubernetes/dashboard/wiki/Integrations) guide.
+and graphs to be available. Read more about it in [Integrations](docs/contributors/Integrations) guide.
 
 ## Documentation
 
-Dashboard documentation can be found on [Wiki](https://github.com/kubernetes/dashboard/wiki) pages, it includes:
+(content in progress)
```

Comment:
This probably shouldn't be in progress when we merge into master.

Reply:
Good catch, I can just pull this for the initial merge. My goal was to get a foundation in place then start the clean up as long as we're all aligned on direction.